src/getTime.js

```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const currentTime = str.match(/\b([0-1]\d|2[0-3]):[0-5]\d\b/);
```

Comment:
it's not really currentTime. It's just some time extracted form the string. 
Perhaps timeMatches?